spec/compiler/parser/parser_spec.cr

```diff
@@ -174,6 +174,13 @@ module Crystal
     assert_syntax_error "def foo!=; end", "unexpected token: !="
     assert_syntax_error "def foo?=(x); end", "unexpected token: ?"
 
+    # 5856
+    assert_syntax_error "def foo=(a,b); end", "setter method 'foo=' cannot receive more than one argument"
```

Comment:
`s/receive/have/`. "receive" is more associated to the call site. Same for the others. Just my opinion.

Reply:
seems reasonable. `have` is a better hint at something static too.